byon/files/temporal-catalog.txt

```diff
@@ -0,0 +1,368 @@
+brooklyn.catalog:

```

Comment:
You are right, it was there for testing other things, I'm sorry. I'm going to remove it.